APIReference.md

```diff
@@ -402,3 +335,33 @@ return a promise.</p>
 | logLevel | <p>an winston log level</p> |
 | authHeaderName |  |
 
+<a name="createStargateUri"></a>
+
+## createStargateUri(baseUrl, baseAuthUrl, keyspace, username, password, logLevel) ⇒
+<p>Create a JSON API connection URI while connecting to Open source JSON API</p>
+
+**Kind**: global function  
+**Returns**: <p>URL as string</p>  
+
+| Param |
+| --- |
+| baseUrl | 
+| baseAuthUrl | 
+| keyspace | 
+| username | 
+| password | 
+| logLevel | 
```

Comment:
Would be great to have a list of valid values for `logLevel`

Reply:
Sure, added in [81c072a](https://github.com/stargate/stargate-mongoose/pull/140/commits/81c072ac15a105f85c0c4fa51da4dffc1f282eda).